pkg/flare/config_check.go

```diff
@@ -63,8 +63,14 @@ func GetConfigCheck(w io.Writer, withDebug bool) error {
 		} else {
 			fmt.Fprintln(w, fmt.Sprintf("%s: %s", color.BlueString("Source"), color.RedString("Unknown provider")))
 		}
-		for i, inst := range c.Instances {
-			fmt.Fprintln(w, fmt.Sprintf("%s %s:", color.BlueString("Instance"), color.CyanString(strconv.Itoa(i+1))))
+		for _, inst := range c.Instances {
+			var ID string
+			if len(c.Instances) == 1 {
```

Comment:
why not generate the ID all the time ?

Reply:
Because checks that are supposed to have nly one instance, like memory doesn't have any generated ID, they just use their name